src/node_contextify.cc

```diff
@@ -289,9 +289,11 @@ void ContextifyContext::WeakCallback(
 void ContextifyContext::WeakCallbackCompileFn(
     const WeakCallbackInfo<CompileFnEntry>& data) {
   CompileFnEntry* entry = data.GetParameter();
-  entry->env->id_to_function_map[entry->id].Reset();
-  entry->env->id_to_function_map.erase(entry->id);
-  delete entry;
+  if (entry->env->compile_fn_entries.erase(entry) != 0) {
+    entry->env->id_to_function_map[entry->id].Reset();
+    entry->env->id_to_function_map.erase(entry->id);
+    delete entry;
```

Comment:
Should `entry` be erased from `env->compile_fn_entries`? I assume so, right?

Reply:
Yes, that is done in the `if` statement itself.

Reply:
Heh, sorry I missed that :)